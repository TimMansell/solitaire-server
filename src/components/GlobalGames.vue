<template>
  <div>
    Games:
    <Counter data-test="global-stats" :number="globalGameCount" />
  </div>
</template>

<script>
import { mapGetters } from 'vuex';
import Counter from '@/components/Counter.vue';

export default {
  name: 'GlobalGames',
  components: {
    Counter,
  },
  computed: {
    ...mapGetters(['globalGameCount']),
  },
};
</script>
