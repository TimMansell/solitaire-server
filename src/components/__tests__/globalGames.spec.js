import { shallowMount } from '@vue/test-utils';
import GlobalGames from '@/components/GlobalGames.vue';

describe('GlobalGames.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(GlobalGames, {
      computed: {
        globalGameCount: () => 1,
      },
    });

    expect(wrapper).toMatchSnapshot();
  });
});
