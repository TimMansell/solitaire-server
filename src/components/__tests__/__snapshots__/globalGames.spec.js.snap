// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalGames.vue matches snapshot 1`] = `
<div>
  Games:
  <counter-stub number="1" data-test="global-stats"></counter-stub>
</div>
`;
